docs/usability/creating_invocable.md

```diff
@@ -0,0 +1,685 @@
+# Creating An Invocable
+
+Whether you spell it invokable or invocable, the idea of an Invocable is central to Stuart.
+If you're unfamiliar with what it is, refer to the using document in the root docs folder or feature_invocable in the features folder.
+In a nutshell, an invokable is a small python file that gets the project Mu environment setup for it.
```

Comment:
this is no longer Project Mu focused.  I have cleaned up the code but not the docs.  Lets not continue to make more Project Mu references